default-config.toml

```diff
@@ -55,6 +55,12 @@ color = true
 # Whether ANSI color codes from the page content should be rendered
 ansi = true
 
+# Whether or not to support source code highlighting in preformatted blocks based on alt text
+highlight_code = true
+
+# Which highlighting style to use (see https://xyproto.github.io/splash/docs/)
+highlight_style = "vim"
```

Comment:
Upon some testing, I find the vim is pretty difficult to read, especially that blue for comments. I find that monokai is a vast improvement and a good default. Let me know if you disagree, but otherwise I'd like to switch it.

Reply:
Sure, I like monokai and it's a good default. Source highlighting themes are intensely personal!